src/draw/lv_draw_buf_private.h

```diff
@@ -0,0 +1,53 @@
+/**
+ * @file lv_draw_buf_private.h
+ *
+ */
+
+#ifndef LV_DRAW_BUF_PRIVATE_H
+#define LV_DRAW_BUF_PRIVATE_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*********************
+ *      INCLUDES
+ *********************/
+
+#include "lv_draw_buf.h"
+
+/*********************
+ *      DEFINES
+ *********************/
+
+/**********************
+ *      TYPEDEFS
+ **********************/
+
+struct lv_draw_buf_handlers_t {
+    lv_draw_buf_malloc_cb buf_malloc_cb;
```

Comment:
all of the function pointer type names are incorrect here. they should at a minimum have an `_t` on the end. If there is no `user_data` associated with these then the type names should end in `xcb_t`

Reply:
True, but it's independent of this PR. Let treat it separately.